assets/containerisableComponents/contributionsThankYou/directDebitDetails.jsx

```diff
@@ -0,0 +1,44 @@
+// @flow
```

Comment:
Why are you creating a new component?

Reply:
It's not new exactly, it's just been moved to a separate file so I could give it some new `PropTypes`. This was the most straightforward way I could see to deal with type errors caused by the DD props.